tests/functional/lib/fx-desktop.js

```diff
@@ -7,6 +7,10 @@ define([
   'intern/chai!assert'
 ], function (assert) {
 
+  function getRemote(context) {
+    return context.get ? context : context.remote;
```

Comment:
Would this also work as `return context.remote || context`?

Although I can see how you arrive at this test based on the refactored code below, it's a little bit surprising/mysterious when I read it in isolation because of the condition part appears unrelated to the return value.